policies/intel-aws-instance-deny-unapproved-instance-types/intel-aws-instance-deny-unapproved-instance-types.sentinel

```diff
@@ -0,0 +1,187 @@
+		output + "\t             i4i.2xlarge, i4i.8xlarge\n",
+		output + "\t             i4i.xlarge, i4i.large\n",
+		output + "\t             i4i.metal, i4i.4xlarge\n",
+		output + "\tGeneral Optimized:\n",
+		output + "\t             m6i.12xlarge,m6i.16xlarge\n",
+		output + "\t             m6i.24xlarge, m6i.32xlarge\n",
+		output + "\t             m6i.2xlarge, m6i.8xlarge\n",
+		output + "\t             m6i.xlarge, m6i.large\n",
+		output + "\t             m6i.metal, m6i.4xlarge\n",
+		output + "\t             m6id.12xlarge, m6id.16xlarge\n",
+		output + "\t             m6id.24xlarge, m6id.32xlarge\n",
+		output + "\t             m6id.2xlarge, m6id.8xlarge\n",
+		output + "\t             m6id.xlarge, m6id.large\n",
+		output + "\t             m6id.metal, m6id.4xlarge\n",
+		output + "\t             m6idn.12xlarge, m6idn.16xlarge\n",
+		output + "\t             m6idn.24xlarge, m6idn.32xlarge\n",
+		output + "\t             m6idn.2xlarge, m6idn.8xlarge\n",
+		output + "\t             m6idn.xlarge, m6idn.large\n",
+		output + "\t             m6idn.4xlarge, m6in.12xlarge\n",
+		output + "\t             m6in.16xlarge, m6in.24xlarge\n",
+		output + "\t             m6in.32xlarge, m6in.2xlarge\n",
+		output + "\t             m6in.8xlarge, m6in.xlarge\n",
+		output + "\t             m6in.large, m6in.4xlarge\n",
+		output + "\tMemory Optimized:\n",
+		output + "\t             r6i.12xlarge,r6i.16xlarge\n",
+		output + "\t             r6i.24xlarge, r6i.32xlarge\n",
+		output + "\t             r6i.2xlarge, r6i.8xlarge\n",
+		output + "\t             r6i.xlarge, r6i.large\n",
+		output + "\t             r6i.metal, r6i.4xlarge\n",
+		output + "\t             r6id.12xlarge, r6id.16xlarge\n",
+		output + "\t             r6id.24xlarge, r6id.32xlarge\n",
+		output + "\t             r6id.2xlarge, r6id.8xlarge\n",
+		output + "\t             r6id.xlarge, r6id.large\n",
+		output + "\t             r6id.metal, r6id.4xlarge\n",
+		output + "\t             r6idn.12xlarge, r6idn.16xlarge\n",
+		output + "\t             r6idn.24xlarge, r6idn.32xlarge\n",
+		output + "\t             r6idn.2xlarge, r6idn.8xlarge\n",
+		output + "\t             r6idn.xlarge, r6idn.large\n",
+		output + "\t             r6idn.4xlarge, r6in.12xlarge\n",
+		output + "\t             r6in.16xlarge, r6in.24xlarge\n",
+		output + "\t             r6in.32xlarge, r6in.2xlarge\n",
+		output + "\t             r6in.8xlarge, r6in.xlarge\n",
+		output + "\t             r6in.large, r6in.4xlarge\n",
+		output + "\t             x2idn.16xlarge, x2idn.24xlarge\n",
+		output + "\t             x2idn.32xlarge, x2idn.metal\n",
+		output + "\t             x2iedn.16xlarge, x2iedn.24xlarge\n",
+		output + "\t             x2iedn.32xlarge, x2iedn.2xlarge\n",
+		output + "\t             x2iedn.8xlarge x2iedn.xlarge\n",
+		output + "\t             x2iedn.metal, x2iedn.4xlarge\n",
+		output + "\tAccelerated Computing:\n",
+		output + "\t             trn1.32xlarge, trn1.2xlarge\n",
+	)
+	return null
+}
```

Comment:
We should look at moving all of the values we are evaluating to a markdown file in a folder for the policy. Shift the output values to providing a link to what we are evaluating so that way we aren't having to maintain the list in print statements and also apply formatting to each line item.

Reply:
Agreed - targeting for next sprint